benches/bench.rs

```diff
@@ -149,6 +149,84 @@ fn bench_add_large_with_string(c: &mut Criterion) {
     });
 }
 
+fn bench_set_multi_small_strings(c: &mut Criterion) {
+    let rt = Runtime::new().unwrap();
+
+    c.bench_function("set_multi_small_strings", |b| {
+        b.to_async(&rt).iter_custom(|iters| async move {
+            let mut client = setup_client().await;
+            let kv = vec![("key1", "value1"), ("key2", "value2"), ("key3", "value3")];
+            let start = std::time::Instant::now();
+            for _ in 0..iters {
+                let _ = client.set_multi(kv.clone(), None, None).await;
+            }
+            start.elapsed()
+        });
+    });
+}
+
+fn bench_set_multi_large_strings(c: &mut Criterion) {
+    let rt = Runtime::new().unwrap();
+
+    c.bench_function("set_multi_large_strings", |b| {
+        b.to_async(&rt).iter_custom(|iters| async move {
+            let mut client = setup_client().await;
+            let large_payload = "a".repeat(LARGE_PAYLOAD_SIZE);
+            let kv = vec![
+                ("key1", &large_payload),
+                ("key2", &large_payload),
+                ("key3", &large_payload),
+            ];
+            let start = std::time::Instant::now();
+            for _ in 0..iters {
+                let _ = client.set_multi(kv.clone(), None, None).await;
+            }
+            start.elapsed()
+        });
+    });
+}
+
+fn bench_set_multi_with_100_large_string_values(c: &mut Criterion) {
```

Comment:
Dropped this in as a "worst case" benchmark for our use-case.  Not necessarily applicable to all, but it should give a good estimation of perf for higher demand workloads.

Reply:
cool you add the output of the new benchmarks either to the PR description or as a comment on the PR. It is nice to see the data without having to pull / run it all